helm-chart/eoapi/templates/services/multidim/hpa.yaml

```
@@ -0,0 +1,53 @@
+{{- if and .Values.multidim.enabled .Values.multidim.autoscaling.enabled }}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: multidim-hpa-{{ .Release.Name }}
+  labels:
+    app: multidim-{{ .Release.Name }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: multidim-{{ .Release.Name }}
+  minReplicas: {{ .Values.multidim.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.multidim.autoscaling.maxReplicas }}
+  behavior:
+    {{- with .Values.multidim.autoscaling.behaviour }}
```

```suggestion
    {{- with .Values.multidim.autoscaling.behavior }}
```
to match with the spelling in the spec